services/autorust/azure-autorust/src/main.rs

```diff
@@ -14,52 +14,63 @@ struct Args {
     /// Generate the publish GitHub workflows
     #[clap(long)]
     publish: bool,
+
+    /// Specify specific package to generate
```

Comment:
Nit: we probably should mention that this takes a list of packages.

Reply:
I added to the help that `Multiple accepted.`
```
USAGE:
    azure-autorust.exe [OPTIONS]

OPTIONS:
    -h, --help                 Print help information
    -p, --package <PACKAGE>    Specify specific package to generate. Multiple accepted
        --publish              Generate the publish GitHub workflows     
```